pkg/manager/manager.go

```diff
@@ -41,35 +41,41 @@ const (
 type VirtletManager struct {
 	server *grpc.Server
 	// libvirt
-	connTool  *libvirttools.ConnectionTool
-	imageTool *libvirttools.ImageTool
+	libvirtConnTool           *libvirttools.ConnectionTool
+	libvirtImageTool          *libvirttools.ImageTool
+	libvirtVirtualizationTool *libvirttools.VirtualizationTool
 	// etcd
-	keysAPITool *etcdtools.KeysAPITool
-	sandboxTool *etcdtools.SandboxTool
+	etcdKeysAPITool *etcdtools.KeysAPITool
+	etcdImageTool   *etcdtools.ImageTool
+	etcdSandboxTool *etcdtools.SandboxTool
 }
 
 func NewVirtletManager(libvirtUri string, poolName string, storageBackend string, etcdEndpoint string) (*VirtletManager, error) {
-	connTool, err := libvirttools.NewConnectionTool(libvirtUri)
+	libvirtConnTool, err := libvirttools.NewConnectionTool(libvirtUri)
 	if err != nil {
 		return nil, err
 	}
-	imageTool, err := libvirttools.NewImageTool(connTool.Conn, poolName, storageBackend)
+	libvirtImageTool, err := libvirttools.NewImageTool(libvirtConnTool.Conn, poolName, storageBackend)
 	if err != nil {
 		return nil, err
 	}
+	libvirtVirtualizationTool := libvirttools.NewVirtualizationTool(libvirtConnTool.Conn)
 	// TODO(nhlfr): Use many endpoints of etcd.
-	keysAPITool, err := etcdtools.NewKeysAPI([]string{etcdEndpoint})
+	etcdKeysAPITool, err := etcdtools.NewKeysAPI([]string{etcdEndpoint})
 	if err != nil {
 		return nil, err
 	}
-	sandboxTool := etcdtools.NewSandboxTool(keysAPITool.KeysAPI)
+	etcdImageTool := etcdtools.NewImageEtcdTool(etcdKeysAPITool.KeysAPI)
+	etcdSandboxTool := etcdtools.NewSandboxTool(etcdKeysAPITool.KeysAPI)
 
 	virtletManager := &VirtletManager{
-		server:      grpc.NewServer(),
-		connTool:    connTool,
-		imageTool:   imageTool,
-		keysAPITool: keysAPITool,
-		sandboxTool: sandboxTool,
+		server:                    grpc.NewServer(),
+		libvirtConnTool:           libvirtConnTool,
+		libvirtImageTool:          libvirtImageTool,
+		libvirtVirtualizationTool: libvirtVirtualizationTool,
+		etcdKeysAPITool:           etcdKeysAPITool,
```

Comment:
Hmmm... Is this prefix needed in entry name? Values provide implementation, but as it's composeable, maybe someday we will have other than based on etcd tools there? This is probably why You provided a struct with common name [there](https://github.com/Mirantis/virtlet/pull/13/files#diff-3fa1b3884d9211562318b8a6453862e4R35) instead of specific one. (btw. this looks like good place to introduce interfaces in place of structs).